plugins/org.jkiss.dbeaver.ext.oceanbase/src/org/jkiss/dbeaver/ext/oceanbase/mysql/model/OceanbaseMySQLCatalog.java

```diff
@@ -66,9 +66,11 @@ protected JDBCStatement prepareChildrenStatement(JDBCSession session, OceanbaseM
                                 procedure == null ? null : JDBCUtils.escapeWildCards(session, procedure.getName()), "%")
                         .getSourceStatement();
             } else {
-                String queryFunctionString = "select * from mysql.proc where db='%s' and type='FUNCTION' and name='%s'";
-                return session
-                        .prepareStatement(String.format(queryFunctionString, owner.getName(), procedure.getName()));
+                String queryFunctionString = "select * from mysql.proc where db=? and type='FUNCTION' and name=?";
+                JDBCPreparedStatement statement = session.prepareStatement(String.format(queryFunctionString, owner.getName(), procedure.getName()));
```

Comment:
Leftover and unnecessary `String.format`

Reply:
Sorry, I missed this. I'll just inline `queryFunctionString`.